src/components/Paginator/Paginator.js

```diff
@@ -111,14 +111,14 @@ const PageButton = styled(Button)`
   ${props => {
```

Comment:
Poderia ter uma desestruturação apenas no início do styled desestruturando o `isActive` e o `theme`. Aí aqui poderia ficar `${isActive && css`background-color: ${theme.colors.primary.main}; color: #fff;`}`  e mais abaixo no :hover poderia ficar só `background-color: ${theme.colors.primary.main}`
O que acha?